moveit_core/kinematic_constraints/include/moveit/kinematic_constraints/utils.h

```diff
@@ -136,7 +136,7 @@ moveit_msgs::Constraints constructGoalConstraints(const std::string& link_name,
  *
  * @return A full constraint message containing both constraints
  */
-moveit_msgs::Constraints constructGoalConstraints(const std::string& link_name, const geometry_msgs::PoseStamped& pose,
+moveit_msgs::msg::Constraints constructGoalConstraints(const std::string& link_name, const geometry_msgs::PoseStamped& pose,
```

Comment:
Same here
```suggestion
moveit_msgs::msg::Constraints constructGoalConstraints(const std::string& link_name, const geometry_msgs::msg::PoseStamped& pose,
```

Reply:
outside scope of this PR

Reply:
Note this approach is fundamentally different from the one we're following to get a minimum viable prototype (going module by module and ensuring that changes compile and make sense from a ROS2 perspective). Anyhow, contributions are valid and will save us time. 

Thanks for putting time into this.